### ISSUE TITLE
Error while interacting with slider button

### ISSUE THREAD

**Issue by MrMathur** [johnwalley/d3-simple-slider#153]
While following the instructions in the documentation for slider-setup, the following error emerges. 

`Uncaught TypeError: Cannot read property 'x' of null
    at SVGGElement.<anonymous> (d3-simple-slider:2)
    at Dispatch.apply (d3.v5.js:796)
    at customEvent (d3.v5.js:1602)
    at gesture (d3.v5.js:1967)
    at SVGGElement.mousedowned (d3.v5.js:1890)
    at SVGGElement.<anonymous> (d3.v5.js:1526)
`

This error emerges when interacting with the slider to change value.

**Comment by janicekchen** [comment]
I just updated to newest version of D3 and it started working for me again!

**Comment by johnwalley** [comment]
Hi! I made a mistake when publishing the new prelease v2.0 which meant you would get that rather than the stable `1.8.0`. I've since updated the npm tags to ensure this doesn't happen

<img width="775" alt="Screenshot 2020-09-10 at 14 28 16" src="https://user-images.githubusercontent.com/981531/92736289-0ab9ee00-f372-11ea-88c5-f2c397a92313.png">

@janicekchen you're correct that upgrading to d3 v6 would fix it as `v.2.0.0-rc.1` is intended to be the version that supports d3 v6.

Apologies for the inconvenience @MrMathur!
